resteasy-client/src/main/java/org/jboss/resteasy/client/jaxrs/ClientHttpEngineBuilder43.java

```diff
@@ -75,6 +83,7 @@ public ClientHttpEngine build()
       {
          SSLConnectionSocketFactory sslsf = null;
          SSLContext theContext = that.sslContext;
+         Iterator clientConfigProviderIterator = ServiceLoader.load(ClientConfigProvider.class).iterator();
```

Comment:
@asoldano in most scenarios it would work, but it is possible to have keystore configured without truststore as well. For example in elytron client we have a test configurations that use key-store but not truststore, eg. here: https://github.com/wildfly-security/wildfly-elytron/blob/1.x/auth/client/src/test/resources/org/wildfly/security/auth/client/test-wildfly-config-v1_5.xml#L51 .